tasks/us/census/tiger.py

```diff
@@ -664,10 +666,10 @@ def populate(self):
         session = current_session()
         stmt = ('INSERT INTO {output} '
                 'SELECT geoid, ST_Union(ST_MakePolygon(ST_ExteriorRing(the_geom))) AS the_geom, '
-                '       MAX(aland) AS aland, cdb_observatory.FIRST(name) AS name '
+                '       MAX(aland) AS aland '
```

Comment:
I reviewed the history of that file, and what changes John and Michelle made. Looks like they went back and forth adding name and removing it and for what I understood looks like this one was a missing part of that changes that are not needed anymore. You could check it looking at the history of the tiger.py class